Cloud slice

Thvnd3r opened the pull request:
-added document for Azure capacity limitations

Thvnd3r committed:
New document for Azure Capacity Limitations

Added document for Azure Capacity Limitations

Thvnd3r committed:
Azure Capacity Limitation update

Cleaned up text

Thvnd3r committed:
azure resource capacity limitations

-added steps for submitting support request
-added screenshots for submitting support request

Thvnd3r committed:
azure resource capacity limitations

-fixed typos

Thvnd3r committed:
Update azure-capacity-limitations.md

Thvnd3r committed:
azure resource capacity limitations

-changed screenshot for deployment model

Thvnd3r committed:
Merge branch 'cloud-slice' of https://github.com/dillonreese/guides into cloud-slice

Thvnd3r committed:
azure resource capacity limitations

-updated screenshot

Thvnd3r committed:
azure resource capacity limitations

-updated deployment model screenshot

Thvnd3r committed:
Update Azure Capacity Limitations.md

Thvnd3r committed:
IDL2 cheat sheet

-added syntax for inline code block
-added syntax for fenced code block
-removed IDL2 Quick Reference Guide title
-added additional syntax for bold, italic and horizontal line

Thvnd3r committed:
Merge branch 'cloud-slice' of https://github.com/dillonreese/guides into cloud-slice

Thvnd3r committed:
Update idl2-quick-reference-guide.md

Thvnd3r committed:
Update idl2-quick-reference-guide.md

Thvnd3r committed:
Update Azure Capacity Limitations.md

Thvnd3r committed:
Delete idl2-quick-reference-guide.md

KirkMunro commented:
Dillon, in this PR, the document you added isn't being linked to from anywhere. Can you also update cloud-slice.md in the root of the cloud-slice folder such that it has a section describing the need and linking to your new doc, with placeholders for AWS/GCP like you will find in the first two sections in that document?

Also, rename "Azure Capacity Limitations.md" as "azure-capacity-limitations.md", and add links in that file that allow you to navigate back to the main file (see how this is done in microsoft-azure/cloud-slice-setup.md with Return to the Cloud Slice Guide links. You can optionally use Back to Top links if they seem worth it, just to follow the UX set up in the rest of this guide.

Thvnd3r committed:
Rename Azure Capacity Limitations.md to azure-capacity-limitations.md

Thvnd3r committed:
cloud-capacity-limitations

-changed name of azure-capacity-limitations file
-added section to cloud slice guide about resource cap limitations
-linked cloud slice guide and resource cap documents together

Thvnd3r committed:
Rename cloud-capacity-limitations.md to azure-capacity-limitations.md

Thvnd3r committed:
Update cloud-slice.md

Thvnd3r committed:
Update cloud-slice.md

Thvnd3r commented:
Kirk,

The requested changes have been made. Please let me know if any other changes are needed.

KirkMunro committed:
Minor update of resource limitations information

Flushed out the wording a little in the new section

KirkMunro committed:
removed unnecessary reference link

KirkMunro committed:
replaced/corrected use of reference links

KirkMunro committed:
Update cloud-slice.md

KirkMunro committed:
added link back to guide to end of document

KirkMunro committed:
fixed link label

KirkMunro reviewed (approved):
LGTM
